### ISSUE TITLE
🐛: Tasks can indeed reference models

### ISSUE THREAD

**Issue by luciansmith** [reproducible-biomedical-modeling/CRBM-Viz#4590]
### Contact Details

lpsmith@uw.edu

### Location

runBiosimulations

### What happened?

Uploaded 
[Figure10.zip](https://github.com/biosimulations/biosimulations/files/10181088/Figure10.zip)
and it complained about tasks referencing models.  This is an incorrect validation error: tasks can indeed reference models, and sometimes *must* reference models, especially when the task involves multiple models.

https://run.biosimulations.org/runs/63912ccf6686da037dc226df#tab=log

### What made it happen?

Uploaded the omex file.

### Version

Production (Deployed on a .org domain)

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
`/root/archive.omex` is not a valid COMBINE/OMEX archive.
  - The SED-ML file at location `Figure10.sedml` is invalid.
    - Data generator `plot_0_0_0` is invalid.
      - Variable `task10b_____time` is invalid.
        - Variable should not reference a model.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jonrkarr** [comment]
The validation would need to be changed in biosimulators_utils.

**Comment by luciansmith** [comment]
Looks like I can't move the issue (though I biosimulators_utils appears to be in a completely different organization?); if you'd like me to recreate it there if it can't be moved, I can do that.

**Comment by jonrkarr** [comment]
I think it would be helpful to organize issues closely to their sources. 

Issues can only be moved within organizations.

**Comment by luciansmith** [comment]
I'm surprised that biosimulations and biosimulators aren't the same organization, but I suppose that's what seemed best at the time.  I'll create a new issue.

**Comment by luciansmith** [comment]
Moved to https://github.com/biosimulators/Biosimulators_utils/issues/124
